packages/xf86driproto.rb

```diff
@@ -0,0 +1,17 @@
+require 'package'
+
+class Xf86driproto < Package
+  description 'The protocols for the X window system provide extended functionality for communication between a X client and the server.'
+  homepage 'https://x.org'
+  version '2.1.1'
+  source_url 'https://www.x.org/archive/individual/proto/xf86driproto-2.1.1.tar.gz'
+  source_sha256 '18ff8de129b89fa24a412a1ec1799f8687f96c186c655b44b1a714a3a5d15d6c'
+
+  def self.build
+    system "./configure --prefix=#{CREW_PREFIX}"
```

Comment:
I noticed there was one file in `/usr/local/lib`, so we probably need `--libdir=#{CREW_LIB_PREFIX}` in this package as well.  See below:
```
$ crew files xf86driproto
(i) xf86driproto: The protocols for the X window system provide extended functionality for communication between a X client and the server.
https://x.org
version 2.1.1
/usr/local/include/X11/dri/xf86dri.h
/usr/local/include/X11/dri/xf86driproto.h
/usr/local/include/X11/dri/xf86dristr.h
/usr/local/lib/pkgconfig/xf86driproto.pc
Total found: 4
```